src/lib/main/core/body/td-core.directive.ts

```diff
@@ -61,16 +75,23 @@ export class TdCoreDirective implements Td, OnInit, OnDestroy {
 				this.cd.detectChanges();
 				break;
 			}
+			case 'change': {
+				this.element.classList.add(`${GRID_PREFIX}-change`);
+
+				const link = this.cellService.build('body', this.column, 'change');
+				link(this.viewContainerRef, this);
+				this.cd.markForCheck();
+				this.cd.detectChanges();
+				break;
+			}
 			default:
 				throw new AppError('td.core', `Invalid mode ${value}`);
 		}
 	}
 
 	get value() {
-		const { column, row, rowIndex, columnIndex } = this;
-		return this.$view.body.render.getValue(row, column, rowIndex, columnIndex);
+		return this.actualValue;
```

Comment:
It looks like no need to persist value in the own field 
just use
`this.changes && this.changes.currentValue`